CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### main
 
+__Improvements__
+- Added @discardableResult to allow developers to choose whether or not certain functions should return a result ([#385](https://github.com/parse-community/Parse-Swift/pull/385)), thanks to [Damian Van de Kauter](https://github.com/vdkdamian).
```

Comment:
You can remove the `@` here. If someone's GitHub name is `@discardableResult` you will be linking them to the changelog